permits/views.py

```diff
@@ -305,8 +298,20 @@ def get_amend_form(self, data=None):
         ):
             # Only set the `status` default value if it's submitted for validation, to prevent accidentally resetting
             # the status
+
+            first_wot_type = (
+                services.get_works_object_type_choices(self.permit_request)
+                .first()
```

Comment:
And if there is mutliple names?

Reply:
It's juste a automated suggestion, that has to be accepted or edited by pilot